xml_to_json.py

```diff
@@ -0,0 +1,86 @@
+"""
+File: xml_to_json.py
+Author: Keshav Balaji
+Description: Standalone Python script to parse an xml file of annotations and write
+the data to a corresponding json file
```

Comment:
A usage example is useful here to show what it expects as inputs defined at the command line